src/main/g8/src/main/scala/$package$/args/Cmd.scala

```diff
@@ -0,0 +1,12 @@
+package $package$.args
+
+/** For scoping commands within args */
+sealed trait Cmd
```

Comment:
Had to rename to `Cmd` because `Command` is already reserved by `decline` :cry:

Reply:
Shrug city.